tests/integration-all/s3/service/utils.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function getMarkers(functionName) {
+  return {
+    start: `--- START ${functionName} ---`,
+    end: `--- END ${functionName} ---`,
```

Comment:
Wouldn't it be better if there's new line after START and before END?

Reply:
Will update it.